.github/workflows/release.yml

```
@@ -155,6 +155,7 @@ jobs:
         uses: docker/build-push-action@v2
         with:
           push: true
+          platforms: "linux/amd64,linux/arm64,linux/armhf"
```

I added `armv7` (common SBCs like raspberry pi or odroid). Also, `"` are not needed
```suggestion
          platforms: linux/amd64, linux/arm64, linux/armhf, linux/armv7
```